SConstruct

```diff
@@ -108,7 +108,7 @@ for x in sorted(glob.glob("platform/*")):
         platform_exporters.append(platform_name)
     if os.path.exists(x + "/api/api.cpp"):
         platform_apis.append(platform_name)
-    if detect.can_build():
+    if os.getenv("GODOT_CAN_BUILD_" + x.removeprefix("platform/").upper()) or detect.can_build():
```

Comment:
```suggestion
    if os.getenv("GODOT_CAN_BUILD_" + platform_name.upper()) or detect.can_build():
```
To handle Windows `\\` prefix as well.

Reply:
Yep I'll make sure to do these suggestions :)

Reply:
done